aiopg/utils.py

```diff
@@ -61,98 +52,77 @@ def gi_code(self):
     def __next__(self):
         return self.send(None)
 
-    @asyncio.coroutine
-    def __iter__(self):
-        resp = yield from self._coro
+    async def __iter__(self):
```

Comment:
`__iter__` should be a regular function, not `async def` one.
The line looks suspicious

Reply:
removed